wondrous-app/utils/helpers.ts

```diff
@@ -92,17 +92,18 @@ export const hasCreateTaskPermission = ({ userPermissionsContext, orgId = undefi
   return permissions.some((i) => [PERMISSIONS.FULL_ACCESS, PERMISSIONS.CREATE_TASK].includes(i));
 };
 
-export const transformTaskToTaskCard = (task, extraData) => ({
+export const transformTaskToTaskCard = (task, extraData = null) => ({
   ...task,
-  assigneeUsername: task?.assignee?.username,
-  assigneeProfilePicture: task?.assignee?.profilePicture,
+  assigneeUsername: task?.assignee?.username || null,
```

Comment:
Why do we need to set this to null?

Reply:
We need to set `null` to override the value of `assigneeUsername` and `assigneeProfilePicture` when the `assignee` is removed.